### PR TITLE
dts: bindings: series of fix typo in multiple directories

### PR DESCRIPTION
**Pull request by ndrs-pst**
I've made typo corrections in various dts/bindings directories:

- `dts/bindings/(adc|arm)/*`
- `dts/bindings/(bluetooth|can|dac|display)/*`
- `dts/bindings/clock/*`
- `dts/bindings/dma/*`
- `dts/bindings/(ethernet|gpio|i2c|interrupt-controlle)/*`
- `dts/bindings/(net|power-domain|pwm|qspi)/*`
- `dts/bindings/pinctrl/*`
- `dts/bindings/(retained_mem|rng|serial|spi)/*`
- `dts/bindings/sensor/*`
- `dts/bindings/(timer|usb-c|usb|watchdog)/*`

Aiming to improve Zephyr project's documentation quality. 🚀

### CODE REVIEW

**Review by henrikbrixandersen — DISMISSED** [review]
Thanks!

File: dts/bindings/watchdog/intel,adsp-watchdog.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ properties:
     type: int
     description: |
       Clock frequency used by counter in Hz. You can specify a frequency here or specify a clock
-      using the clocks propertie.
+      using the clocks properties.
```
Comment (ajarmouni-st) [inline dts/bindings/watchdog/intel,adsp-watchdog.yaml:19]:
```suggestion
      using the property "clocks".
```

File: dts/bindings/watchdog/snps,designware-watchdog.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ properties:
     type: int
     description: |
       Clock frequency used by counter in Hz. You can specify a frequency here or specify a clock
-      using the clocks propertie.
+      using the clocks properties.
```
Comment (ajarmouni-st) [inline dts/bindings/watchdog/snps,designware-watchdog.yaml:19]:
```suggestion
      using the property "clocks".
```

File: dts/bindings/serial/st,stm32-uart-base.yaml [hunk 3]
```diff
@@ -90,9 +90,9 @@ properties:
   fifo-enable:
     type: boolean
     description: |
-      Enables transmit and receive FIFO using default FIFO confugration (typically threasholds
+      Enables transmit and receive FIFO using default FIFO configuration (typically thresholds
```
Comment (ajarmouni-st) [inline dts/bindings/serial/st,stm32-uart-base.yaml:93]:
```suggestion
      Enables transmit and receive FIFO using default FIFO configuration (typically threshold is
```

File: dts/bindings/sensor/st,lsm6dsv16x-common.yaml [hunk 4]
```diff
@@ -137,7 +137,7 @@ properties:
       The values are taken in accordance to lsm6dsv16x_data_rate_t enumerative in hal/st
       module. Please note that this values will not change the operating mode, which will remain
```
Comment (ajarmouni-st) [inline dts/bindings/sensor/st,lsm6dsv16x-common.yaml:138]:
```suggestion
      module. Please note that these values will not change the operating mode, which will remain
```

File: dts/bindings/rng/st,stm32-rng.yaml [hunk 5]
```diff
@@ -18,7 +18,7 @@ properties:
          the clock domain used, for instance:
                    <&rcc STM32_SRC_MSI CLK48_SEL(3)> /* RNG clock domain set to MSI */
          A correctly configured domain clock is required to allow the integrated low
-         sampling clock detection mecanism to behave properly.
+         sampling clock detection mechanism to behave properly.
          In provided example, MSI should be configured to provide 48Mhz clock.
```
Comment (ajarmouni-st) [inline dts/bindings/rng/st,stm32-rng.yaml:22]:
```suggestion
         In the provided example, MSI should be configured to provide 48Mhz clock.
```

File: dts/bindings/power-domain/power-domain-gpio-monitor.yaml [hunk 6]
```diff
@@ -6,7 +6,7 @@ description: |
 
   This power domain monitors the state of a GPIO pin to detect whether a power
   rail is on/off. Therefore, performing resume/suspend on power domain won't
-  change physical state of power rails and those action won't be triggerd on
+  change physical state of power rails and those action won't be triggered on
```
Comment (ajarmouni-st) [inline dts/bindings/power-domain/power-domain-gpio-monitor.yaml:9]:
```suggestion
  change physical state of power rails and that action won't be triggered on
```

File: dts/bindings/power-domain/power-domain-gpio-monitor.yaml [hunk 7]
```diff
@@ -6,7 +6,7 @@ description: |
...
   child nodes. Additionally, due to the asynchronous nature of monitoring a
```
Comment (ajarmouni-st) [inline dts/bindings/power-domain/power-domain-gpio-monitor.yaml:10]:
```suggestion
  child nodes. Additionally, due to the asynchronous nature of monitoring, a
```

File: dts/bindings/net/wireless/gpio-radio-coex.yaml [hunk 8]
```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 description: |
-    Generic representation of Coexistance pin interface for radios. This
+    Generic representation of Coexistence pin interface for radios. This
     interface is usually available on Wifi/Bluetooth/LTE modules to
     interact with each other when sharing same antenna. This prevents
```
Comment (ajarmouni-st) [inline dts/bindings/net/wireless/gpio-radio-coex.yaml:7]:
```suggestion
    interact with each other when sharing the same antenna. This prevents
```

File: dts/bindings/interrupt-controller/cypress,psoc6-intmux.yaml [hunk 9]
```diff
@@ -60,7 +60,7 @@ description: |
   The interrupt can be enabled/disable at NVIC at line 20 as usual.
 
   Notes:
-  1) Multiple definitions will generate multiple interrutps
+  1) Multiple definitions will generate multiple interrupts
   2) The interrupt sources are shared between Cortex-M0+/M4. These means, can
      trigger action in parallel in both processors.
```
Comment (ajarmouni-st) [inline dts/bindings/interrupt-controller/cypress,psoc6-intmux.yaml:65]:
```suggestion
  2) The interrupt sources are shared between Cortex-M0+/M4. This means they
     can trigger actions in parallel on both processors.
```

File: dts/bindings/i2c/atmel,sam-i2c-twim.yaml [hunk 10]
```diff
@@ -14,7 +14,7 @@ description: |
 
   When using speeds above standard mode, user may need adjust clock and data
   lines slew and strength parameters.  In general, slew 0 and minimal strength
-  is enougth for short buses and light loads.  As reference, the below
+  is enough for short buses and light loads.  As reference, the below
```
Comment (ajarmouni-st) [inline dts/bindings/i2c/atmel,sam-i2c-twim.yaml:17]:
```suggestion
  is enough for short buses and light loads.  As a reference, the below
```

File: dts/bindings/dma/st,stm32-dma-v2.yaml [hunk 11]
```diff
@@ -16,7 +16,7 @@ description: |
     2. slot: DMA periph request ID, which is written in the DMAREQ_ID of the DMAMUX_CxCR
     this value is 0 for Memory-to-memory transfers
     or a value between <1> .. <dma-generators> (not supported yet)
-    or a value beweeen <dma-generators>+1  ..  <dma-generators>+<dma-requests>
+    or a value between <dma-generators>+1  ..  <dma-generators>+<dma-requests>
     3. channel-config: A 32bit mask specifying the DMA channel configuration
     A name custom DMA flags for channel configuration is used
     which is device dependent see stm32_dma.h:
```
Comment (ajarmouni-st) [inline dts/bindings/dma/st,stm32-dma-v2.yaml:22]:
```suggestion
    which is device dependent. See stm32_dma.h:
```

File: dts/bindings/can/microchip,mcp251xfd.yaml [hunk 12]
```diff
@@ -48,7 +48,7 @@ properties:
     type: boolean
     description: |
       Enables controller PLL, which multiples input clock frequency x10.
```
Comment (ajarmouni-st) [inline dts/bindings/can/microchip,mcp251xfd.yaml:50]:
```suggestion
      Enables controller PLL, which multiplies input clock frequency by 10.
```

File: dts/bindings/bluetooth/ambiq,bt-hci-spi.yaml [hunk 13]
```diff
@@ -27,6 +27,6 @@ properties:
     type: phandle-array
     description: |
       This clkreq gpio is used to send the XO32MHz clock request to host from
-      from controller. The host needs to enable XO32MHz when receiveing low to
-      high edge interrupt and disable XO32MHz when receiveing high to low edge
+      from controller. The host needs to enable XO32MHz when receiving low to
+      high edge interrupt and disable XO32MHz when receiving high to low edge
       interrupt.
```
Comment (ajarmouni-st) [inline dts/bindings/bluetooth/ambiq,bt-hci-spi.yaml:32]:
```suggestion
      controller. The host needs to enable XO32MHz when receiving low to high
      edge interrupts and disable XO32MHz when receiving high to low edge
      interrupts.
```

**Comment by ndrs-pst** [comment]
@ajarmouni-st, thank you for the additional suggestion.
I will push another commit that reflects your suggestion.

**Review by npal-cy — DISMISSED** [review]
Thanks!

**Review by ajarmouni-st — DISMISSED** [review]

**Review by fabiobaltieri — DISMISSED** [review]

File: dts/bindings/clock/st,stm32-rcc.yaml [hunk 14]
```diff
@@ -54,7 +54,7 @@ description: |
   Domain clock is independent from the bus/gatted clock and allows access to the device's
```
Comment (kartben) [inline dts/bindings/clock/st,stm32-rcc.yaml:54]:
```suggestion
  Domain clock is independent from the bus/gated clock and allows access to the device's
```

File: dts/bindings/interrupt-controller/cypress,psoc6-intmux.yaml [hunk 15]
```diff
@@ -60,9 +60,9 @@ description: |
   The interrupt can be enabled/disable at NVIC at line 20 as usual.
```
Comment (kartben) [inline dts/bindings/interrupt-controller/cypress,psoc6-intmux.yaml:60]:
```suggestion
  The interrupt can be enabled/disabled at NVIC at line 20 as usual.
```

**Review by kartben — COMMENTED** [review]
Very nice, thanks!
Caught a couple more that were showing up around the actual changes in your initial PR.

**Review by kartben — CHANGES_REQUESTED** [review]
Meant to make this a request for change sorry! And I hope you don't mind :)

**Comment by ndrs-pst** [comment]
> Meant to make this a request for change sorry! And I hope you don't mind :)

The changes have been incorporated and squashed into this commit:  https://github.com/zephyrproject-rtos/zephyr/pull/68194/commits/31d05cd606d794831b39df128c44a29a8cf1d25b

**Review by kartben — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Review by ifyall — APPROVED** [review]

**Pull request merged**